modules/installation-special-config-raid.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-raid_{context}"]
+== Configuring a RAID-enabled data volume
+
+You can enable software RAID partitioning to provide an external data volume. {product-title} supports RAID 0, RAID 1, RAID 4, RAID 5, RAID 6, and RAID 10 for software backup, data protection, and fault tolerance. See "About disk mirroring" for more details.
+
+. Create a Butane config that uses software RAID to configure partitioning for two or more disks.
+
+* To configure mirrored boot disks that use RAID 1, create a `$HOME/clusterconfig/raid1-storage.bu` file, for example:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.8.0
+metadata:
+  name: raid1-storage
+  labels:
+    machineconfiguration.openshift.io/role: worker
+boot_device:
+  mirror:
+    devices:
+      - /dev/sda
+      - /dev/sdb
+storage:
+  disks:
+    - device: /dev/sda
+      partitions:
+        - label: root-1
+          size_mib: 25000 <1>
+        - label: var-1
+    - device: /dev/sdb
+      partitions:
+        - label: root-2
+          size_mib: 25000 <1>
+        - label: var-2
+  raid:
+    - name: md-var
+      level: raid1
+      devices:
+        - /dev/disk/by-partlabel/var-1
+        - /dev/disk/by-partlabel/var-2
+  filesystems:
+    - device: /dev/md/md-var
+      path: /var
+      format: xfs
+      wipe_filesystem: true
+      with_mount_unit: true
+----
+<1> When adding a data partition to the boot disk, a minimum value of 25000 mebibytes is recommended. The root file system is automatically resized to fill all available space up to the specified offset. If no value is specified, or if the specified value is smaller than the recommended minimum, the resulting root file system will be too small, and future reinstalls of {op-system} might overwrite the beginning of the data partition.
+
+* To configure secondary disks that use RAID 1, create a `$HOME/clusterconfig/raid1-alt-storage.bu` file, for example:
++
+[source,yaml]
+----
+variant: openshift
+version: 4.8.0
+metadata:
+  name: raid1-alt-storage
+  labels:
+    machineconfiguration.openshift.io/role: worker
+storage:
+  disks:
+    - device: /dev/sdb
+      wipe_table: true
+      partitions:
+        - label: var-1
+    - device: /dev/sdc
+      wipe_table: true
+      partitions:
+        - label: var-2
```

Comment:
- I'd suggest changing the devices here to `/dev/sdc` and `/dev/sdd`, to avoid any conflicts from the boot disk example.

- I'd suggeset changing the labels to `data-1` and `data-2` as well, to avoid any conflict with the boot disk example

Reply:
Good call! Resolved.